### PR TITLE
Migration logging to cloudwatch - RHCLOUD-4284

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

**Comment by tahmidefaz** [comment]
We only need `INFO` level logging for `sqlalchemy.engine` when running migrations, hence the requirement for the `configure_migrations_logging()`. 

**Question:** I moved over the `console` handler from the place where it was set up earlier. Do we really need it? I think we should be fine with `logstash` here, unless I am missing something.

File: logconfig.yaml [hunk 1]
```diff
@@ -44,9 +51,15 @@ handlers:
     formatter: logstash
     filters:
     - contextual
+  console:
```
Comment (jharting) [inline logconfig.yaml:54]: What is this handler for?

File: logconfig.yaml [hunk 2]
```diff
@@ -44,9 +51,15 @@ handlers:
...
+    class: logging.StreamHandler
+    formatter: generic
+    stream: ext://sys.stderr
 formatters:
   human_readable:
     format: "[%(asctime)s] [%(process)d] [%(thread)d] [%(name)s] [%(levelname)s] %(message)s"
+  generic:
+    format: "%(levelname)-5.5s [%(name)s] %(message)s"
```
Comment (jharting) [inline logconfig.yaml:62]: Do we two formats?

File: Makefile [hunk 3]
```diff
@@ -7,7 +7,7 @@ test:
 	pytest --cov=.
 
 upgrade_db:
-	python manage.py db upgrade
+	MIGRATIONS_LOG_LEVEL=INFO python manage.py db upgrade
```
Comment (jharting) [inline Makefile:10]: That's the default value, right?

File: logconfig.yaml (hunk 1 above)
Comment (tahmidefaz) [reply logconfig.yaml:54]: It was being used by the migrations earlier, and I moved it here from the old  `alembic.ini`. I don't think we need it, it'd be better to use `logstash` imo. What do you think?

File: logconfig.yaml (hunk 2 above)
Comment (tahmidefaz) [reply logconfig.yaml:62]: Yes, the migrations was using this format earlier, we can switch this to the `human_readable` format as well

File: logconfig.yaml (hunk 1 above)
Comment (jharting) [reply logconfig.yaml:54]: Agreed, let's have it produce logs in the same format than the rest

File: Makefile (hunk 3 above)
Comment (tahmidefaz) [reply Makefile:10]: Yes, it doesn't make a difference atm. I put it there to increase visibility.

File: logconfig.yaml (hunk 1 above)
Comment (tahmidefaz) [reply logconfig.yaml]: done! :+1:

File: logconfig.yaml (hunk 2 above)
Comment (tahmidefaz) [reply logconfig.yaml]: removed :+1:

**Comment by jharting** [comment]
@tahmidefaz I changed this to work differently. Take a look at https://github.com/RedHatInsights/insights-host-inventory/pull/675/commits/f577b63673cd4fb602798ad4c99eca7bf4319e48 when you're back

**Pull request merged**